pages/common/progress.md

```diff
@@ -0,0 +1,12 @@
+# progress
+
+> Show progress of an ongoing system command. Supported commands include: cp, mv, dd, tar, bsdtar, cat, rsync, scp, grep, fgrep, egrep, cut, sort, md5sum, sha1sum, sha224sum, sha256sum, sha384sum, sha512sum, adb, gzip, gunzip, bzip2, bunzip2, xz, unxz, lzma, unlzma, 7z, 7za, zip, unzip. 
+> More information: <https://github.com/Xfennec/progress>.
+
+- Show progress of ongoing commands, updates every 2 seconds:
+
+`watch progress`
+
+- Show progress of ongoing commands, includes estimate of remaining time:
+
+`watch progress -w`
```

Comment:
watch != progress

Reply:
yes, that is what I meant because it would keep updating the output of progress. However, for consistency, we can remove watch such that this does not involve calling other commands...

This should be fine:

    progress -w